tests/annotations/tests.py

```diff
@@ -130,6 +130,21 @@ def test_joined_annotation(self):
         for book in books:
             self.assertEqual(book.num_awards, book.publisher.num_awards)
 
+    def test_joined_transformed_annotation(self):
+        Employee.objects.create(
+            first_name='John', last_name='Doe', age=18, store=self.s1, salary=15000)
+        Employee.objects.create(
+            first_name='Jane', last_name='Jones', age=30, store=self.s2, salary=30000)
+        Employee.objects.create(
+            first_name='Jo', last_name='Smith', age=55, store=self.s3, salary=50000)
+
+        employees = Employee.objects.annotate(store_opened_year=F('store__original_opening__year'))
```

Comment:
Can we add a test for using related field without any transform?

Reply:
Can we a add test for a custom transform, e.g. `__abs`?
```python
with register_lookup(IntegerField, Abs):
    ...
```

Reply:
I think the test for using a related field without a transform already exists - isn't that what `test_joined_annotation` does?

Reply:
> I think the test for using a related field without a transform already exists - isn't that what `test_joined_annotation` does?

Ahh, yes :+1: